packages/server/lib/screenshots.coffee

```diff
@@ -313,14 +314,24 @@ getPath = (data, ext, screenshotsFolder) ->
   else
     names = [data.titles.map(replaceInvalidChars).join(RUNNABLE_SEPARATOR)]
 
+  # truncate file names to be less than 220 characters
+  # to accomodate filename size limits
+  maxFileNameLength = 220
```

Comment:
This is a bit funky. From testing outside of Cypress, seems like this value should be 255 - length of `(failed) (N).png`, but that still causes a name too long error on CI. That doesn't happen when this value is a bit smaller 🤷‍♀️